Document on resend-code API changes for mobile claim update (#3090)
wso2/docs-is: en/docs/guides/identity-lifecycles/enable-verification-for-updated-mobile-number.md

    -   An SMS OTP verification is not triggered if the mobile number to be updated is the same as the previously verified mobile number of the user.
    -   Sending the SMS OTP verification is skipped in the following instances:
        1. The `verifyMobile` claim is not set to true in the SCIM 2.0 request.

>         2. The claim update is invoked by a user other than the claim owner or a non privileged user.

Reviewer: ```suggestion
        2. The claim update is invoked by a user other than the claim owner or a non-privileged user.
```

        enable_verification = true
        verification_sms_otp_validity = "5"
        ```

>     3. By default mobile number verification is not allowed for privileged users. Add the following property to the above `deployment.toml` config to enable this server wide.

Reviewer: ```suggestion
    3. By default, mobile number verification is not allowed for privileged users. Add the following property to the above `deployment.toml` config to enable this server wide.
```

---

> Additionally can send the following curl command to resend a new SMS OTP code by a privileged user.

Reviewer: ```suggestion
Additionally, you can use the following curl command to resend a new SMS OTP code by a privileged user.
```
